src/navigation/Navigator.tsx

```diff
@@ -56,13 +60,25 @@ const styles = (headerHeight?: number) =>
 
 const Stack = createStackNavigator<RoutesParams>()
 
-const Navigator = (): JSX.Element => {
-  const [isPressed, setIsPressed] = React.useState<boolean>(false)
-  const [isHomeButtonPressed, setIsHomeButtonPressed] = React.useState<boolean>(false)
+const Navigator = (): JSX.Element | null => {
+  const [isPressed, setIsPressed] = useState<boolean>(false)
+  const [isHomeButtonPressed, setIsHomeButtonPressed] = useState<boolean>(false)
+  const [waitingForInit, setWaitingForInit] = useState<boolean>(true)
+  const [introShown, setIntroShown] = useState<boolean>(false)
 
   // Set only height for tablets since header doesn't scale auto
   const headerHeight = useTabletHeaderHeight(wp('15%'))
 
+  useEffect(() => {
+    const initialize = async () => {
+      const professions = await AsyncStorage.getSelectedProfessions()
+      setIntroShown(professions !== null)
```

Comment:
Is it required that a profession gets selected? Maybe it would be more explicit if we saved a separate value `introShown` to the async storage.

I also think that if I don't want to select a profession I should not have to click on `Überspringen` each time I open the app (this fails the alternate course of the issue).

Reply:
Yes, currently profession is null, when the value was never set. If you skip the selection, an empty array should be saved, therefore the alternate course should work. But I forgot to set this. 

If you think the clearness of a separate value will outweigh that one more slow reading from AsyncStorage we need then, I will add it. Let me know what you prefer.

Reply:
As discussed the current way is fine.